src/Rating.php

```diff
@@ -39,13 +43,20 @@ public function getRates($originPostalCode, $postalCode, $weight, array $options
             ],
         ];
 
+        if (!empty($options['service_codes'])) {
+            $content['services']['service-code'] = $this->parseServiceCodes($options);
+        }
+
         if (!empty($options['option_codes'])) {
             $content['options']['option'] = $this->parseOptionCodes($options);
         }
```

Comment:
**OPTIONAL CHANGE REQUEST**

Now that we're adding more options I think we should be splitting the options for Canada Post from the options for Guzzle. They can either be separate variables or the Canada Post options can be within a sub-array keyed by `canada_post` (or the other way around).

This will provide more clarity and prevent possible conflicts in option names (unlikely to happen but still).

Not sure how are you standing with time with this, you can leave this for later.